src/Dfe.PlanTech.Web/Views/Recommendations/RecommendationContent.cshtml

```diff
@@ -21,7 +23,16 @@
                             await Html.RenderPartialAsync("Components/PageComponentFactory", content);
                         }
                     </div>
+                    @if (headerWithContent is RecommendationChunk chunk && !string.IsNullOrEmpty(chunk.CSUrl) && !string.IsNullOrEmpty(chunk.CSLinkText))
+                    {
+                        <div class="govuk-!-margin-top-3 govuk-!-margin-bottom-9">
+                            <a class="govuk-link govuk-!-font-weight-bold" href=@chunk.CSUrl target="_blank">
+                                @chunk.CSLinkText
```

Comment:
The font size for the C&S Link doesn't quite match the prototype, its too small, which was surprising with it being the same classes as the share link below

Looks like the font size is overriden when `govuk-body` is also present to 19 - but to be on the safe side I'd just add `govuk-body` as a class to the a tag, that way it stays consistent with the share link below it

Reply:
[79c3dfe](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/718/commits/79c3dfe3847af48f2a9bb99c54f9d0e7137c7df6)

Reply:
FWIW the share link is wrapped in a `p` element, which sets the font-size to 19px.